doc/source/whatsnew/v0.19.0.txt

```diff
@@ -323,6 +323,22 @@ Other enhancements
                        index=['row1', 'row2'])
      df.sort_values(by='row2', axis=1)
 
+- ``Index`` now supports ``.dropna`` to exclude missing values (:issue:`6194`)
+
+  .. ipython:: python
+
+     idx = pd.Index([1, 2, np.nan, 4])
+     idx.dropna()
+
+For ``MultiIndex``, values are dropped if any level is missing by default. Specifying
+``how='all'`` drops values to all levels are missing.
```

Comment:
" drops values to all levels are missing." -> "only drops values where all levels are missing"